### PR TITLE
Adminping fix w/ server config option

### PR DESCRIPTION
**Pull request by AnywayFarus**
## About The Pull Request

Normal return of sound notifications about active tickets. After consulting with GoldenAlpharex, I decided to add the ability to configure this, with the mode obviously turned off.

## How This Contributes To The Skyrat Roleplay Experience

There is less chance that the ticket/opfor will be ignored, which will probably help to create an interesting situation faster, or solve the moment that the RP situation was violated.

## Changelog

:cl: Vishenka0704
add: add option to enable/disable ticket ping in server config
fix: oppfor and ticket ping now works
/:cl:

### CODE REVIEW

**Comment by AnywayFarus** [comment]
oh.. my mistake

File: modular_skyrat/modules/subsystems/code/ticket_ping/ticket_ss.dm [hunk 1]
```diff
@@ -6,6 +6,8 @@ SUBSYSTEM_DEF(ticket_ping)
 	wait = 3 MINUTES
 
 /datum/controller/subsystem/ticket_ping/fire(resumed)
+	if(!CONFIG_GET(flag/ticket_ping))
```
Comment (Zonespace27) [inline modular_skyrat/modules/subsystems/code/ticket_ping/ticket_ss.dm]: this is bad, just set it on Initialize()

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/adminhelp.dm [hunk 2]
```diff
@@ -251,6 +251,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	ticket_interactions = list()
 	player_interactions = list()
 
+	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 3 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping
```
Comment (GoldenAlpharex) [inline code/modules/admin/verbs/adminhelp.dm]: This time duration should be done via config too, so it's easy for keyholders to change that duration without having to make it a PR.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/adminhelp.dm (hunk 2 above)
Comment (AnywayFarus) [reply code/modules/admin/verbs/adminhelp.dm]: yep yep, it ready

File: code/modules/admin/verbs/adminhelp.dm [hunk 3]
```diff
@@ -251,7 +251,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	ticket_interactions = list()
 	player_interactions = list()
 
-	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 3 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping
+	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 2 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping | this is not responsible for the notification itself, but only for adding the ticket to the list of those to notify.
```
Comment (GoldenAlpharex) [inline code/modules/admin/verbs/adminhelp.dm:254]: So why change it to 2 minutes here, exactly? Won't that cause the first ping to happen after two minutes? Or is it additional time before the first ping?

File: modular_skyrat/modules/subsystems/code/ticket_ping/ticket_ss.dm [hunk 4]
```diff
@@ -1,4 +1,4 @@
-/// Pings admins every 3 minutes for all open tickets/OPFOR applications
+/// Pings admins every (time choisen in config) for all open tickets/OPFOR applications
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/subsystems/code/ticket_ping/ticket_ss.dm]:
```suggestion
/// Pings admins every (time chosen in config) for all open tickets/OPFOR applications
```

File: modular_skyrat/master_files/code/_globalvars/configuration.dm [hunk 5]
```diff
@@ -46,4 +46,4 @@ GLOBAL_VAR_INIT(looc_allowed, TRUE)
 
 /datum/config_entry/flag/admin_event_uses_chaos
 
-/datum/config_entry/flag/ticket_ping
+/datum/config_entry/number/ticket_ping_frequency
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/_globalvars/configuration.dm:50]:
```suggestion
/// Ticket ping frequency. Set 0 for disable that subsystem. 3000 - 5 minutes, 600 - 1 minute.
/datum/config_entry/number/ticket_ping_frequency
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/adminhelp.dm (hunk 3 above)
Comment (AnywayFarus) [reply code/modules/admin/verbs/adminhelp.dm:254]: This is exclusively the time when the ticket will be added to the system with a ping.
The pings themselves only happen when the ping subsystem fires

**Review by GoldenAlpharex — APPROVED** [review]
Looks good, thank you!

**Pull request merged**